modules/channel-web/src/backend/api.ts

```diff
@@ -247,7 +246,7 @@ export default async (bp: typeof sdk, db: Database) => {
   })
 
   function validateUserId(userId: string) {
-    if (!userId || userId.length > USER_ID_MAX_LENGTH || userId.length < USER_ID_MIN_LENGTH) {
+    if (!userId || userId.length > USER_ID_MAX_LENGTH) {
```

Comment:
As @slvnperron  suggested can you make sure !== Undefined

Reply:
@EFF already handled by `!userId`. I added the `userId.toLowerCase() === 'undefined'` just to be extra safe